salt/metalk8s/salt/minion/running.sls

```diff
@@ -16,3 +16,7 @@ Ensure salt-minion running:
     - enable: True
     - require:
       - module: Wait until salt-minion restarted
+  module.run:
+    - test.ping: []
```

Comment:
Didn't we decide to use `- name: test.ping`?

Reply:
No we use 
```
use_superseded:
  - module.run
```
So we can't